src/connector/src/sink/formatter/mod.rs

```diff
@@ -92,6 +92,9 @@ pub enum SinkFormatterImpl {
     AppendOnlyTemplate(AppendOnlyFormatter<TemplateEncoder, TemplateEncoder>),
     AppendOnlyTextTemplate(AppendOnlyFormatter<TextEncoder, TemplateEncoder>),
     AppendOnlyBytesTemplate(AppendOnlyFormatter<BytesEncoder, TemplateEncoder>),
+    AppendOnlyBytes(AppendOnlyFormatter<JsonEncoder, BytesEncoder>),
+    AppendOnlyTextBytes(AppendOnlyFormatter<TextEncoder, BytesEncoder>),
+    AppendOnlyBytesBytes(AppendOnlyFormatter<BytesEncoder, BytesEncoder>),
```

Comment:
```suggestion
    AppendOnlyBytes(AppendOnlyFormatter<BytesEncoder, BytesEncoder>),
```
Similarly, it makes no sense to allow key encode of text or json either.